client/src/app.jsx

```diff
@@ -1,20 +1,78 @@
-import React, { Component } from 'react';
-import './app.css';
+import React from 'react';
+import axios from 'axios';
```

Comment:
I like that you used Axios instead of Jquery.

Reply:
Thanks Justin!